content/6_Advanced/Eulers_Formula.mdx

```diff
@@ -11,91 +11,536 @@ frequency: 1
 
 ## Introduction
 
-<IncompleteSection />
+A planar graph is a graph that can be drawn on a plane without any edges crossing. In other words, it can be embedded in the plane such that no two edges intersect except at their endpoints.
+
+<Resources>
+    <Resource
+        source="MIT"
+        title="MIT Course Notes"
+        url="https://dspace.mit.edu/bitstream/handle/1721.1/104426/6-042j-spring-2010/contents/readings/MIT6_042JS10_chap12.pdf"
+
+    >
+        6.024J course notes
+    </Resource>
+    <Resource
+        source="Wiki"
+        title="Planar Graph"
+        url="https://en.wikipedia.org/wiki/Planar_graph"
+    >
+        Wiki Definition
+    </Resource>
+    <Resource
+        source="Rutgers"
+        title="Planar Graph"
+        url="https://sites.math.rutgers.edu/~sk1233/courses/graphtheory-F11/planar.pdf"
+    />
+
+</Resources>
+
+## Euler's Formula
+
+Euler's Formula states that any correct embedding of a connected planar graph satistfies:-
+
+    <center>     $$ V − E + F = 2 $$  </center>
+
+$$ V = Vertices $$
+
+$$ E = Edges $$
+
+$$ F = Faces $$
```

Comment:
do you...check how this looks in the markdown renderer

Reply:
Yeah I will change it.

Reply:
Any other changes?

Reply:
we'll see